src/ApparentHorizons/Tags.hpp

```diff
@@ -326,5 +327,22 @@ struct SurfaceIntegral : db::ComputeTag {
                                    StrahlkorperTags::Strahlkorper<Frame>>;
 };
 
+struct Area : db::SimpleTag {
+  static std::string name() noexcept { return "Area"; }
+  using type = double;
+};
+
+template <typename Frame>
+struct AreaCompute : Area, db::ComputeTag {
```

Comment:
Add the following comment:

/// Computes the surface area of a Strahlkorer, \f$A = \oint_S dA\f$ given an `AreaElement` \f$dA\f$ and a `Strahlkorper` \f$S\f$.